src/base/bittorrent/session.cpp

```diff
@@ -1783,10 +1783,12 @@ void Session::processShareLimits()
                         if (m_maxRatioAction == Remove) {
                             logger->addMessage(tr("'%1' reached the maximum ratio you set. Removed.").arg(torrent->name()));
                             deleteTorrent(torrent->hash());
+                            continue; // Avoid possible crash in "torrent->name()" when processing seeding time
```

Comment:
Please move "continue" outside the "if-else" block since it's a common part.

Reply:
Done